modules/avs/spec/models/avs/v0/after_visit_summary_spec.rb

```diff
@@ -11,7 +11,8 @@
         'generatedDate' => '2020-01-01T00:00:00Z',
         'data' => {
           'patientInfo' => {
-            'icn' => '1234567890V123456'
+            'icn' => '1234567890V123456',
+            'smokingStatus' => 'Current smoker'
```

Comment:
Will there always be a value for `smokingStatus`?  Do we need a test case to account for this being an empty string, or not present at all?

Reply:
good question, it'll be best to assume that it could be empty or not present. I'll update.

Reply:
updated!